app/static/assets/sitemap.xml

```diff
@@ -14,6 +14,10 @@
         <loc>https://www.oppiafoundation.org/about</loc>
         <priority>0.80</priority>
     </url>
+    <url>
+        <loc>https://www.oppiafoundation.org/about_oppia</loc>
```

Comment:
Hm, I seem to recall that we were advised to use hyphens in URLs instead of underscores (for SEO) -- see https://github.com/oppia/oppia/issues/6772 

If that is the case, maybe we should change this here and elsewhere to about-oppia? /cc @dchen97

Reply:
I have changed all URL references to use hyphens instead of underscores. Thanks!